packages/build/src/time/aggregate.js

```diff
@@ -108,7 +108,7 @@ const addTypeTimers = function (timers) {
 const TYPE_TIMERS = [
   { name: 'system', stages: ['resolve_config', 'get_plugins_options', 'start_plugins', 'others'] },
   { name: 'plugin', stages: ['load_plugins', 'run_plugins'] },
-  { name: 'user', stages: ['build_command', 'functions_bundling', 'deploy_site'] },
+  { name: 'user', stages: ['build_command', 'edge_functions_bundling', 'functions_bundling', 'deploy_site'] },
```

Comment:
This is entirely unrelated to the PR, but I noticed that edge-functions-bundling does not count toward any of these buckets. So I thought it might make sense to count it towards the `user` bucket. Is this okay?